core/src/main/scala/cats/data/NonEmptyVector.scala

```diff
@@ -216,6 +216,9 @@ private[data] sealed trait NonEmptyVectorInstances {
       override def foldRight[A, B](fa: NonEmptyVector[A], lb: Eval[B])(f: (A, Eval[B]) => Eval[B]): Eval[B] =
         fa.foldRight(lb)(f)
 
+      override def get[A](fa: NonEmptyVector[A])(idx: Long): Option[A] =
+        fa.get(idx.toInt)
```

Comment:
In general I wonder if it's worth just using `Int` as the index. I can't think of many structures you can index into with `Long` values that you'd also want to fold over. (But that doesn't mean they don't exist!)